source/agora/crypto/Key.d

```diff
@@ -322,7 +326,7 @@ public struct SecretKey
         import std.format : phobos_format = format;
         import ocean.text.convert.Formatter : ocean_format = format;
         assert(phobos_format("%s", sd) == "**SECRET**");
-        assert(ocean_format("{}", sd) == "**SECRET**");
+        () @trusted { assert(ocean_format("{}", sd) == "**SECRET**"); }();
```

Comment:
Why is the `ocean_format` nested in `@trusted`?

Reply:
Because `format` isn't `@safe`, and having the `@safe` at the top also applies to `unittest`.

Reply:
Is `phobos_format` `@safe`? I can't find `@safe` in the prototype of the `format` function in `std.format`.